src/main/java/mate/academy/AsyncRequestProcessor.java

```diff
@@ -1,16 +1,36 @@
 package mate.academy;
 
+import java.util.Map;
 import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.Executor;
+import java.util.concurrent.TimeUnit;
 
 public class AsyncRequestProcessor {
+    private static final long TIMEOUT = 1000;
```

Comment:
The TIMEOUT constant is set to 1000 milliseconds. Ensure that this value is appropriate for your use case. If the expected processing time is less, consider lowering it to improve responsiveness.